### PR TITLE
Temporary fix: change "github" to "GitHub"

### PR DESCRIPTION
**Pull request by DilumAluthge**
Not sure if this is sufficient.

@StefanKarpinski

### CODE REVIEW

File: src/register.jl [hunk 1]
```diff
@@ -624,6 +624,7 @@ function register(
     if !isempty(package_repo)
         package_repo = GitTools.normalize_url(package_repo)
     end
+    package_repo = replace(package_repo, "github" => "GitHub")
```
Comment (StefanKarpinski) [inline src/register.jl:627]:
```suggestion
    package_repo = replace(package_repo, r"^https?://github\.com/"i => "https://GitHub.com/")
```
Don't want to change "github" to "GitHub" elsewhere in the URL.

**Comment by GunnarFarneback** [comment]
This seems premature. Isn't it better to hold off on merging into General until we know how the download API issues will be resolved?

**Comment by DilumAluthge** [comment]
Sure, seems reasonable to me.

**Comment by DilumAluthge** [comment]
Actually, if I understand correctly, we can probably turn AutoMerge back on for new versions of existing packages. Since those PRs don't make changes to URLs. It's just new package PRs that we have to keep disables for now.

**Comment by GunnarFarneback** [comment]
I would suspect that all registration requests will be rejected at this point because they are changing the registered URL.

**Comment by DilumAluthge** [comment]
> I would suspect that all registration requests will be rejected at this point because they are changing the registered URL.

Ahhh I see.

**Comment by KristofferC** [comment]
Github seems fixed now so don't think this is needed.

**Pull request closed without merging**